ironic-config/ironic.conf.j2

```diff
@@ -1,32 +1,27 @@
 [DEFAULT]
-{% if "HTTP_BASIC_HTPASSWD" in env and env.HTTP_BASIC_HTPASSWD | length %}
-auth_strategy = http_basic
-http_basic_auth_user_file = /etc/ironic/htpasswd
-{% else %}
 auth_strategy = noauth
-{% endif %}
 debug = true
-default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
-enabled_bios_interfaces = idrac-wsman,no-bios,redfish,idrac-redfish
-enabled_boot_interfaces = pxe,ipxe,fake,redfish-virtual-media,idrac-redfish-virtual-media
-enabled_deploy_interfaces = direct,fake,ramdisk
+enabled_bios_interfaces = idrac-wsman,no-bios,redfish,idrac-redfish,ilo
+enabled_boot_interfaces = ipxe,ilo-ipxe,pxe,ilo-pxe,fake,redfish-virtual-media,idrac-redfish-virtual-media,ilo-virtual-media
+enabled_deploy_interfaces = direct,fake,ramdisk,custom-agent
 # NOTE(dtantsur): when changing this, make sure to update the driver
 # dependencies in Dockerfile.
-enabled_hardware_types = ipmi,idrac,irmc,fake-hardware,redfish,manual-management
-enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish
-enabled_management_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,noop
-enabled_power_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish
-enabled_raid_interfaces = no-raid,irmc,agent,fake,idrac-wsman
-enabled_vendor_interfaces = ipmitool,no-vendor,idrac,fake
+enabled_hardware_types = ipmi,idrac,irmc,fake-hardware,redfish,ibmc,manual-management,ilo,ilo5
```

Comment:
Do we package proliantutils and the iBMC library (forgot the name)? If not, we cannot enable these.

Reply:
I had the same doubt and actually wanted to ask if we want to add them, but we can discuss this separately.
I will remove ilo and ibmc for the time being